### DISCUSSION TITLE
Is it possible to save layouts?

### DISCUSSION THREAD

**Discussion by SimonBrandner** [hyprwm/Hyprland#1424] [Q&A]
Is there a feature similar to [the layout saving feature in i3wm](https://i3wm.org/docs/layout-saving.html) that would allow me to save the positions of windows in the tabbed layout and load them when starting my PC again the next day?

**Comment by vaxerski** [comment]
no

**Reply by SimonBrandner** [reply]
Should I open a feature request?

**Reply by vaxerski** [reply]
I'm not your dad, do what you want.

**Reply by SimonBrandner** [reply]
:D [Here](https://github.com/hyprwm/Hyprland/issues/1426) it is
